scripts/bmm_examples/Example3_Bays2009.R

```diff
@@ -38,96 +38,98 @@ if (nChains >  parallel::detectCores()) {
-                                 
-                                 # control commands for the sampler
-                                 control = list(adapt_delta = adapt_delta, 
-                                                max_treedepth = max_treedepth))  
-  
-  save(list=ls(), file=here(filename))  
-} else load(here(filename))
+filename <- here('output','fit_bays2009_3p_model')
+
+fit_bays_2009 <- bmm(model = model_bays_2009,
+                     formula = formula_bays_2009, 
+                     data = data_bays_2009, 
+                     
+                     # save settings
+                     sample_prior = TRUE,
+                     save_pars = save_pars(all = TRUE),
+                     
+                     # add brms settings
+                     warmup = warmup_samples,
+                     iter = warmup_samples + postwarmup_samples, 
+                     chains = nChains,
+                     cores = parallel::detectCores(),
+                     
+                     # control commands for the sampler
+                     control = list(adapt_delta = adapt_delta, 
+                                    max_treedepth = max_treedepth),
+                     
+                     # save results to file
+                     file = filename)
 
 #############################################################################!
 # 2) Model evaluation                                                    ####
 #############################################################################!
 
 ## 2.1) fit & summary ----------------------------------------------------------
 # plot the posterior predictive check to evaluate overall model fit
-pp_check(fit_bays2009)
+pp_check(fit_bays_2009)
 
-# print results summary. There is 1 divergent transition, but we will ignore it
+# print resultformula# print results summary. There is 1 divergent transition, but we will ignore it
 # for the purposes of this illustration. For real analyses, follow the suggestions
 # to remove this issue
-summary(fit_bays2009)
+summary(fit_bays_2009)
 
 ## 2.2) Extract parameter estimates --------------------------------------------
 # extract the fixed effects from the model and determine the rows that contain
 # the relevant parameter estimates
-fixedEff <- fixef(fit_bays2009)
+fixedEff <- fixef(fit_bays_2009)
 thetat <- fixedEff[grepl("thetat",rownames(fixedEff)),]
 thetant <- fixedEff[grepl("thetant",rownames(fixedEff)),]
 kappa <- fixedEff[grepl("kappa_",rownames(fixedEff)),]
 
 # transform parameters because brms uses special link functions
 kappa <- exp(kappa)
 sd <- k2sd(kappa[,1]) 
-pmem <- exp(thetat)/(exp(thetat)+exp(thetant)+exp(0))
-pnt <- exp(thetant)/(exp(thetat)+exp(thetant)+exp(0))
-pg <- exp(0)/(exp(thetat)+exp(thetant)+exp(0))
+pmem <- exp(thetat) / (exp(thetat) + exp(thetant) + exp(0))
+pnt <- exp(thetant) / (exp(thetat) + exp(thetant) + exp(0))
+pg <- exp(0) / (exp(thetat) + exp(thetant) + exp(0))
 
 # print parameter estimates
 kappa
```

Comment:
won't let me comment on the clean_plot function because it's not changed code. Since you define that in an external function file and load, maybe get rid of the clean_plot definition here?

Reply:
Yes, deleted the function here.